cmd/cmd_group.go

```diff
@@ -93,7 +93,7 @@ func createGroup(ctx *cli.Context) error {
 		opts.InitGroupMember = addrList
 	}
 
-	broadcastMode := tx.BroadcastMode_BROADCAST_MODE_BLOCK
+	broadcastMode := tx.BroadcastMode_BROADCAST_MODE_SYNC
```

Comment:
Can we make it global

Reply:
same for txopt, i see most of them are the same

Reply:
updated